src/states/node.rs

```diff
@@ -1141,19 +1107,6 @@ impl Node {
             return Err(RoutingError::FailedSignature);
         }
 
-        if !self.chain.is_member() {
-            debug!(
-                "{} Client {:?} rejected: We are not an established node yet.",
-                self, pub_id
-            );
-            self.send_direct_message(
-                pub_id,
-                DirectMessage::BootstrapResponse(Err(BootstrapResponseError::NotApproved)),
```

Comment:
I am not clear where this has moved or what made it unecessary in EstablishingNode?

Reply:
You are right, this got accidentally lost. I'll put it back.